helm/cilium/templates/default-policies/configmap.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.defaultPolicies.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cilium-default-policies
+  namespace: {{ .Release.Namespace }}
+data:
+  ingress.yaml: |-
+    apiVersion: cilium.io/v2
+    kind: CiliumNetworkPolicy
+    metadata:
+      name: allow-all-ingress-from-host
+      namespace: kube-system
+    spec:
+      endpointSelector: {}
+      ingress:
+      - fromEntities:
+        - host
+        - remote-node
```

Comment:
IMO it would be nice to extract that to a separate file, see https://github.com/giantswarm/cluster-openstack/blob/4ee638c4a9546f73ddd1f8ab075213ad5251e019/helm/cluster-openstack/templates/_helpers.tpl#L46